public_html/wp-content/mu-plugins/blocks/assets/src/shared/avatar-size/index.js

```diff
@@ -51,6 +51,7 @@ function AvatarSizeControl( {
 	onChange,
 	initialPosition,
 	...props
+	// i may just be misunderstanding the spread operator, but it seems like a bad idea to have arbitrary inputs mapped to fixed function params. wouldn't we want the params to be explicitly and unchanging?
```

Comment:
I think I was sort of using `...props` for anything that is optional, while parameters that are explicitly named in the signature are "required".

I'll have to think about this some more, but my initial thought is that there's no harm in including the spread of additional inputs because any values that aren't explicitly used are ignored anyway.

One potential advantage of using `...props` that I can think of is that in a case like this one where a component contains other components, which have their own signatures, is that if any of those components change or are replaced, we don't necessarily have to update the parent component's signature.